src/pages/examples/File.md

```diff
@@ -0,0 +1,43 @@
+This example demonstrates a quirk when using `React` and `redux-form` with File input types.
+
+If you've tried setting up `redux-form` with a File field, you may have encountered a browser error like the following:
+
+* Chrome: `Uncaught DOMException: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string.`
+
+* FireFox: `SecurityError: The operation is insecure.`
+
+* Safari: `Error: InvalidStateError: DOM Exception 11`
+
+Inputs with `type="file"` do not support setting the `value` property programmatically<sup>[1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#File_inputs)</sup>.
+
+To avoid this warning, remove the `value` property from the associated file field present in the `fields` prop that `redux-form` provides to your wrapped component.
+
+As seen in the [Simple](#/examples/simple) example, for most input types it is sufficient to spread the properties passed in by `redux-form`:
```

Comment:
Trying to balance context and terseness here. Perhaps there are improvements to be made here by way of using more consistent vocabulary?